src/pymodaq/examples/qt_less_standalone_module.py

```diff
@@ -33,6 +36,9 @@ def __init__(self, name: str, host: str = "localhost", **kwargs) -> None:
         self._fake_position = 0
         self.start_listen()
         self._stored = []
+        # register DataWithAxes for deserialization
+        cls = DataWithAxes
```

Comment:
I naively thought the registration would be done because I added the registration in the new method of the objects but in fact for this to append, the object should be instantiated at least once. So the question is, before serializing, you will have to create at least once the object and therefore register it so why is it not working?

Reply:
Because serialization and deserialization happen in two different processes:
The serializing part works, but the deserializing module never created such an object, but just gets plain bytes.